### DISCUSSION TITLE
The given data was invalid

### DISCUSSION THREAD

**Discussion by andylarkum** [lucky-media/cloud#16] [Q&A]
I promise, I am very trying! ;)

When trying to edit content (e.g. homepage), when I click "save and publish", I get the toast: "The given data was invalid"

Too new to Statamic to know how to debug this :(

I also swapped out the Cloud logo for my company logo, which displays fine in the footer, but doesn't display in the header?

Any ideas?

Thanks

A.

**Comment by lokmanm** [comment]
Hey @andylarkum,

No worries, usually that error comes when you forget a required field. Make sure to expand the sets in the `blocks` so you can see if a field is missing and it should be displayed with red - indicating that is a required field.

As for the logo it should change both on the footer and header once you change it on the settings. ( Globals -> Theme -> Logo ). If it's not reflecting maybe try to run `php artisan optimize:clear` or try a Hard Refresh on your browser ( CMD / CTRL + Shift + R ).|

Hopefully I can help, let me know if anything else comes up.
